<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_proj</class>
 <widget class="QMainWindow" name="window_proj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proj</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbox_must">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>241</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>必填</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbox_even">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>151</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbox_ampli">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>104</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbtn_run">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>158</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
